### ISSUE TITLE
Missing translatable strings in FAQ and ABOUT

### ISSUE THREAD

**Issue by Cwpute** [funmusicplace/mirlo#759]
Could you add the content of https://mirlo.space/pages/en/faq to translatable strings ? they are quite essential to understanding what Mirlo is about and how it will evolve.
The second paragraoh of the About page is not traslatable either : https://mirlo.space/pages/about

**Comment by MickeyMcc** [comment]
That PR fixes the issue with the second block on the About page not being translatable.

The FAQ page is set up kind of different.  It's shipped as just markdown so is outside of react/i18n - in the page path though you can see that it's scoped as /en for english so when we have translations we could just add /pages/es/faq for spanish, /pages/fr/faq for french etc. as additional .md files/routes.  I think that's reasonable since it's not likely to be much else but static text and it's easier to maintain that sort of formatting in markdown than in react & json.

Any objections to keeping the faq content as a full-text non-dynamic page in markdown and using url paths to manage the translation versions?

**Comment by Cwpute** [comment]
Does that mean translators will have to translate thc FAQ out of Transifex,.or will it rtill be doable inside the tool ?
If not, there should be readily available information on where to find these files and how to translate them. Online documentation with global (and eventually language-specific) translation directions may be necessary at some point.

Le 9 juin 2024 02:04:19 GMT+02:00, Marshall McClenon ***@***.***> a écrit :

**Comment by simonv3** [comment]
Yeah I don't know what best practice is here. To me, because of the complex formatting, it makes sense to keep it in the md, but this way there's no notification system if things get changed / added / etc. Transifex does make a lot of that easier. 

I'd be fine with converting it all into json (a lot of the FAQ already is).

**Comment by Cwpute** [comment]
It seems Weblate provides "partial" support fwr Markdown files (https://docs.weblate.org/en/latest/formats/markdown.html). I don't know what "partial" means here, but perhaps this is worth a look ?
Crowdin on the other side provides full Markdown support, it seems.

Le 9 juin 2024 23:15:17 GMT+02:00, Psi ***@***.***> a écrit :

**Comment by simonv3** [comment]
Made a comment in transifex support forum to hear what people recommend https://community.transifex.com/t/translating-large-paragraphs-of-text/4023
